.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           name: "RSpec"
           command: bundle exec rspec
           environment:
-            SS4_ENDPOINT: "https://scholarsphere.test/api/v1"
+            SS4_ENDPOINT: "https://scholarsphere-rmd-api.dsrd.libraries.psu.edu/api/v1"
```

Comment:
are we to keep this deployment around for testing?

Reply:
We _could_ but we don't have to. I was thinking we'd switch this over to qa.